src/main/java/org/openstreetmap/atlas/checks/base/BaseCheck.java

```diff
@@ -60,7 +60,7 @@
     private final List<String> countries;
     private final Map<String, List<String>> flagLanguageMap;
     // OSM Identifiers are used to keep track of flagged features
-    private final Set<T> flaggedIdentifiers = ConcurrentHashMap.newKeySet();
+    private transient Set<T> flaggedIdentifiers = ConcurrentHashMap.newKeySet();
```

Comment:
Because you are using lazy initialization, I would suggest just setting this initially to null. It will kind of force people to use the method even if they are not thinking about serialization.

Reply:
I agree. Assigning null to the variable.